src/transports/dash/manifest_parser.ts

```diff
@@ -180,7 +180,7 @@ export default function generateManifestParser(
           });
           return processMpdParserResponse(value.continue(resources));
         }
-      }).catch((err) => {
+      }, (err) => {
```

Comment:
Shouldn't it be per-resource? Ex: what if multiple resources (like multiple xlinks) where needed and only one fails?

I think it has more of its place inside the `map` function higher in this file